cf-agent/cf-agent.c

```diff
@@ -102,6 +103,7 @@ extern int PR_NOTKEPT;
 static bool ALLCLASSESREPORT = false; /* GLOBAL_P */
 static bool ALWAYS_VALIDATE = false; /* GLOBAL_P */
 static bool CFPARANOID = false; /* GLOBAL_P */
+static bool PERFORM_DB_CHECK = false;
 
```

Comment:
of what significance is the comment /* GLOBAL_P */ ?

Reply:
Don't know.